src/components/ControlPanel.vue

```diff
@@ -74,15 +94,53 @@ export default {
     socket: Object
   },
   components: {
-    NodesTable
+    NodesTable,
+    DialogAddRemove
   },
   computed: {
-    ...mapGetters(['nodes', 'appInfo'])
+    ...mapGetters(['nodes', 'appInfo', 'zwave'])
+  },
+  watch: {
+    addRemoveEndDate: {
+      immediate: true,
+      handler (newVal) {
+        if (this.addRemoveTimer) {
+          clearInterval(this.addRemoveTimer)
+        }
+        this.addRemoveTimer = setInterval(() => {
+          this.now = new Date()
+
+          const s = Math.trunc((this.addRemoveEndDate - this.now) / 1000)
+          this.addRemoveWorking = s > 0 ? `${this.addRemoveMode} started: ${s}s remaining` : null
+
+          if (this.now > newVal) {
+            this.now = newVal
+            clearInterval(this.addRemoveTimer)
+          }
+        }, 100)
+      }
+    },
+    appInfo: {
+      deep: true,
+      handler (newVal) {
+        if (newVal.controllerStatus.indexOf('clusion') > 0) {
+          this.onAddRemoveStateChange(newVal.controllerStatus)
+        }
+      }
```

Comment:
A cleaner way to do this is to create a `computed` property let's call this `controllerStatus` that returns `this.appInfo.controllerStatus` (or even better call it inclusionStatus and make the checks needed there) and then watch only that :)